src/wallet/wallet.h

```diff
@@ -100,7 +100,7 @@ enum WalletFeature
     FEATURE_HD = 120200,    // Hierarchical key derivation after BIP32 (HD Wallet), BIP44 (multi-coin), BIP39 (mnemonic)
                             // which uses on-the-fly private key derivation
 
-    FEATURE_LATEST = 61000
+    FEATURE_LATEST = FEATURE_HD
```

Comment:
nit, but i think thats something we usually do?

```suggestion
    FEATURE_LATEST = FEATURE_HD,
```

Reply:
Imo, no need.

Reply:
We do it both ways 🙈

Reply:
I also don't see a real need i just brought it up because it was suggested a while ago in one of my PRs (https://github.com/dashpay/dash/pull/3483#discussion_r425439110) and i thought its the practice we want to follow :D wouldn't it make sense to declare either of those as "standard" for the future?

Reply:
Heh, right. I personally prefer to have `,` at the end of line, so that when a new line is added the diff is cleaner. But it's not smth I'm going to fight over :)

Reply:
Hehe alright, i still don't get why we not just declare either of both as practice to follow 🤷‍♂️ since it would be a pretty simple thing to do and it would bring consistency for this case in the future.. but i also don't wanna fight too much for this nit-nit at the end, if you both feel mixing this styles it is fine, lets move on 🙈

Reply:
My whole purpose is to just as close to upstream as reasonably possible. It's probably super trivial